courses/free-courses-pt_BR.md

```diff
@@ -246,6 +246,7 @@
 * [Curso de Node.js](https://www.youtube.com/playlist?list=PLJ_KhUnlXUPtbtLwaxxUxHqvcNQndmI4B) - Victor Lima Guia do Programador (YouTube)
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab (YouTube)
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
+* [Node.js - Fundamentos](https://www.treinaweb.com.br/curso/nodejs-fundamentos) - Wesley Gado (TreinaWeb)
```

Comment:
Looks like this is a paid course. Please add only courses which are free to use

Reply:
@shrn01 updated course link